sdks/java/bom/common.gradle

```diff
@@ -0,0 +1,128 @@
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'org.apache.beam.module'
+apply plugin: 'maven-publish'
+apply plugin: 'java-platform'
+
+javaPlatform {
+    allowDependencies()
+}
+
+def isRelease(Project project) {
+    return project.hasProperty('isRelease')
+}
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            groupId = project.mavenGroupId
+            artifactId = archivesBaseName
+            version = project.version
+            pom {
+                name = project.description
+                if (project.hasProperty("summary")) {
+                    description = project.summary
+                }
+                url = "http://beam.apache.org"
+                inceptionYear = "2016"
+                licenses {
+                    license {
+                        name = "Apache License, Version 2.0"
+                        url = "http://www.apache.org/licenses/LICENSE-2.0.txt"
+                        distribution = "repo"
+                    }
+                }
+                scm {
+                    connection = "scm:git:https://gitbox.apache.org/repos/asf/beam.git"
+                    developerConnection = "scm:git:https://gitbox.apache.org/repos/asf/beam.git"
+                    url = "https://gitbox.apache.org/repos/asf?p=beam.git;a=summary"
+                }
+                issueManagement {
+                    system = "jira"
+                    url = "https://issues.apache.org/jira/browse/BEAM"
+                }
+                mailingLists {
+                    mailingList {
+                        name = "Beam Dev"
+                        subscribe = "dev-subscribe@beam.apache.org"
+                        unsubscribe = "dev-unsubscribe@beam.apache.org"
+                        post = "dev@beam.apache.org"
+                        archive = "http://www.mail-archive.com/dev%beam.apache.org"
+                    }
+                    mailingList {
+                        name = "Beam User"
+                        subscribe = "user-subscribe@beam.apache.org"
+                        unsubscribe = "user-unsubscribe@beam.apache.org"
+                        post = "user@beam.apache.org"
+                        archive = "http://www.mail-archive.com/user%beam.apache.org"
```

Comment:
This URL is 404
You might have an issue around the percent sign

Reply:
Fixed